src/app/page.js

```diff
@@ -1,34 +1,43 @@
 'use client'
 
-import { AiOutlineSearch } from 'react-icons/ai'
+import { useEffect, useState } from 'react'
+
 import { BsStars } from 'react-icons/bs'
-import CollegeResult from './components/CollegeResult'
 import Footer from './components/Footer/Footer'
 import Image from 'next/image'
 import Link from 'next/link'
 import Navbar from './components/Navbar/Navbar'
+import SearchBar from '../components/SearchBar'
 import bacpacTitle from '../assets/bacpacTitle.png'
 import bookImgLogo from '../assets/bookimg.png'
+import client from '../client'
 import discord from '../assets/discordLog.png'
-import universityData from '../../data/university_data'
-import { useState } from 'react'
-import SearchBar from '../components/SearchBar'
+import { gql } from '@apollo/client'
 
+const query = gql`
```

Comment:
Let's make a file and import the query from there. Let's try to maintain a single file which contains all the queries.